### ISSUE TITLE
Sudden Increase in Loss and Corrupted Output Images During Zero123 Model Training

### ISSUE THREAD

**Issue by sunkymepro** [kxhit/zero123-hf#16]
I am training a Zero123-hf model to generate images of a face from an input image and a corresponding pose vector. However, during training, I observe a sudden increase in loss, and the output images become corrupted. Below are the details of my command:

python train_zero1to3.py --train_data_dir /data/root/path/ --pretrained_model_name_or_path ./sd-image-variations-diffusers --train_batch_size 2 --dataloader_num_workers 16 --output_dir logs --gradient_checkpointing --mixed_precision no --resume_from_checkpoint latest

the loss image is shown as below:

![Image](https://github.com/user-attachments/assets/985577ba-1921-4927-a30f-e451614c16ab)

I also observed that when setting the learning rate to 1e-4, the pose of the generated image is correct, but the image quality is relatively poor. The results are shown below (after 6,500 iterations). From left to right, the images are: the input image, the two generated results, and the ground truth.

![Image](https://github.com/user-attachments/assets/1393b97b-b0f3-4609-9503-2b1b9666a3e1)

As the number of iterations increases, the pose of the generated image becomes incorrect, and eventually, the output degrades into a completely black image. The results are shown below.

![Image](https://github.com/user-attachments/assets/2fd225e8-6989-4cca-b9fa-cd17bc381fa5)

(15000 iterations)

![Image](https://github.com/user-attachments/assets/0e0d3b38-bb58-44df-97a5-9ca259ad63da)

(19000 iterations)

I would like to know where my training is going wrong, thank you.

**Comment by kxhit** [comment]
Hi, sorry for the late reply. I don't know exactly what is happening here. It could be some gradients unstable issues or corrupted training data. If you could reproduce this problem in your setting, I would suggest you to check the training data at this step first and then track the gradients/step_loss. Maybe do some grad norm or resume the training before it broken to try to simply fix. 

Hope it helps.
